src/main/kotlin/com/openlattice/transporter/TransporterUtils.kt

```diff
@@ -0,0 +1,110 @@
+package com.openlattice.transporter
```

Comment:
Any chance you can name this class `TransporterSqlUtils` to head off the `Utils` => dumping grounds risk?

Reply:
I think TransporterQueries is probably the right name.